docs/explanations/status_columns.md

```diff
@@ -188,9 +203,9 @@ For example, running `sudo pro status --format=json` on an attached machine may
 ```
 
 Some particularly important attributes in the output include:
-* `attached`: This boolean value indicates whether this machine is attached to an Ubuntu Pro account. This does not tell you if any particular service (e.g. `esm-infra`) is enabled. You must check the individual service item in the `services` list for that status (described below).
+* `attached`: This boolean value indicates whether this machine is attached to an Ubuntu Pro account (Starting with version 28.1, we recommend using the [u.pro.status.is_attached.v1](references/api:u.pro.status.is_attached.v1) API instead). This does not tell you if any particular service (e.g. `esm-infra`) is enabled. You must check the individual service item in the `services` list for that status (described below).
```

Comment:
Since it's not really an aside, I think this is better off as a new sentence rather than being in brackets (same with the one just below at line 208).

Reply:
Addressed